app/models/course.rb

```diff
@@ -3,4 +3,10 @@ class Course < ApplicationRecord
     has_many :students, dependent: :destroy
     has_many :assignments, dependent: :destroy
     has_many :polls, dependent: :destroy
+    validates :semester, presence: true,
+                         inclusion: {in: ["SP", "FA", "sp", "fa"]}
```

Comment:
Should we handle summer semesters?

Reply:
oh good point. I'll add that in, then merge it.